### ISSUE TITLE
Feature: Recommend usage of new routes object

### ISSUE THREAD

**Issue by ConduciveMammal** [Shopify/theme-check#230]
A lot of developers still use the old method of hardcoding various Shopify routes.

Given there's a fairly small amount of routes, could a check be included to ensure liquid routes are being used?
![image](https://user-images.githubusercontent.com/18140157/112333777-cb6e4180-8cb2-11eb-8f7c-6eb6b30635bc.png)

```liquid
<form action="{{ routes.search_url }}">...</form>

Vs.
<form action="/search">...</form>
```

**Comment by sharvy** [comment]
@ConduciveMammal @charlespwd If no one is working on this issue, I want to grab it. :-)

**Comment by charlespwd** [comment]
Absolutely! Let us know if you have any issues and if there are things we could improve in CONTRIBUTING.md

**Comment by ConduciveMammal** [comment]
@sharvy any progress on this?

**Comment by sharvy** [comment]
@ConduciveMammal got busy with some other work 🙈
